srcpkgs/go-ipfs/files/ipfs/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+mkdir -p /ipfs
+mkdir -p /ipns
```

Comment:
These are the default directories for the ipfs mounts, I'm not 100% happy here. We could also take the initialising apart instead of doing it idempotently in the daemon command, which would allow us configure the daemon after initialising it, which in turn enables us to use custom paths here. What would the right place here be? `/var/run/go-ipfs/<ipfs/ipns>` or `/var/lib/go-ipfs/<ipfs/ipns>` is what I'd say, but I'm not 100% sure on this.

Reply:
There is an [optional variable `make_dirs`][1], maybe you can use it.

[1]: https://github.com/void-linux/void-packages/blob/master/Manual.md#optional-variables

Reply:
Nice! Will definitely use that. Still doesn't answer where these mounts should be though...